include/blosc2.h

```diff
@@ -985,6 +985,8 @@ static const blosc2_cparams BLOSC2_CPARAMS_DEFAULTS = {
   (zero) in the fields of the struct is passed to a function.
  */
 typedef struct {
+  int32_t typesize;
+  //!< The type size (8).
```

Comment:
Why `typesize` cannot be read for the blosc2 header?  Probably we discussed this, but could you refresh my memories?

Reply:
Changed in d62e4d8